assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -335,6 +335,12 @@ che.infra.kubernetes.password=
 che.infra.kubernetes.oauth_token=
 che.infra.kubernetes.trust_certs=
 
+# Defines the way how servers are exposed to the world in k8s infra.
+# List of  strategies implemented in Che: default-host, multi-host, single-host
+che.infra.kubernetes.server_strategy=default-host
+# Used to generate domain for a server in a workspace
+che.infra.kubernetes.ingress.domain=unspecified-ingress-domain
```

Comment:
Is it used for each of server strategies?

Reply:
And what are the possible values?

Reply:
> Is it used for each of server strategies?

no, it is used for `multi-host` strategy only

> And what are the possible values?

Something like `192.168.99.101.nip.io`. Really depends on DNS provider, public multi-level DNS, IP, etc.

Reply:
> no, it is used for multi-host strategy only

Then maybe it makes sense to make it more clear in doc or even property name?
Is it possible that default value `unspecified-ingress-domain` will work? If not, maybe it'd better to leave it empty and fail tomcat start if it is not configured?

Reply:
I didn't add this property in code, so not sure whether I should change the name now. But I can add a comment about that.
In a case when multi-host is not chosen strategy failure on start when the value is not set would break Che in different valid cases. Value `unspecified-ingress-domain` won't work but hints a user that he needs to address this issue.

Reply:
> In a case when multi-host is not chosen strategy failure on start when the value is not set would break Che in different valid cases.

I mean failing start only if multi-host is chosen and `che.infra.kubernetes.ingress.domain` is not configured